helm-charts/splunk-otel-collector/templates/configmap-fluentd.yaml

```diff
@@ -221,49 +215,53 @@ data:
         <record>
           # set the sourcetype from splunk.com/sourcetype pod annotation or set it to kube:container:CONTAINER_NAME
           sourcetype ${record.dig("kubernetes", "annotations", "splunk.com/sourcetype") ? "kube:"+record.dig("kubernetes", "annotations", "splunk.com/sourcetype") : "kube:container:"+record.dig("kubernetes","container_name")}
-          container_name ${record.dig("kubernetes","container_name")}
-          namespace ${record.dig("kubernetes","namespace_name")}
-          pod ${record.dig("kubernetes","pod_name")}
-          container_id ${record.dig("docker","container_id")}
-          pod_uid ${record.dig("kubernetes","pod_id")}
-          node_name "#{ENV['K8S_NODE_NAME']}"
-          cluster_name {{ .Values.clusterName }}
```

Comment:
why drop node and cluster names?

Reply:
It'll get attached by the resource processor in the agent now.